pipeline_dash/viz/dash/components/jobs_pipeline_fig.py

```diff
@@ -1,12 +1,15 @@
+import collections
 import itertools
 import time
 import uuid
 from statistics import median
-from typing import Dict, List, Optional
+from typing import Dict, Optional
```

Comment:
idk that it matters all that much, but with 3.10 as the minimum you should be able to drop all `Dict`, `List`, etc types and just use `dict`/`list`

Reply:
yea, probably a holdover of when I first started this project, on py3.7 :joy:

Reply:
Going to overhaul types in a separate PR